src/JB721TiersHookProjectDeployer.sol

```diff
@@ -78,7 +80,7 @@ contract JB721TiersHookProjectDeployer is JBPermissioned, IJB721TiersHookProject
         projectId = DIRECTORY.PROJECTS().count() + 1;
 
         // Deploy the hook.
-        hook = HOOK_DEPLOYER.deployHookFor(projectId, deployTiersHookConfig);
+        hook = HOOK_DEPLOYER.deployHookFor(projectId, deployTiersHookConfig, salt);
```

Comment:
We should also hash it here with the `msg.sender`, this way we make a path using hashes that needs to be followed to get the same deployment address for `721` ([for reference](https://github.com/Bananapus/nana-core/blob/b5dbcdd0a3bad6c8b5830718ee8dcd5094606b84/src/JBController.sol#L462)). Same thing for `launchRulesetsFor` and `queueRulesetsOf`

Reply:
Fixed

Reply:
the line/function didn't change :eyes: